templates/index.html

```diff
@@ -8,5 +8,6 @@
   </head>
   <body>
     <div>Testing</div>
+    <script type="module" src="../client/index.js"></script>
```

Comment:
This path will change in the future.  I'm thinking we can use an environment variable to point at the right script location.